data/GMSim.py

```diff
@@ -0,0 +1,434 @@
+from re import T
```

Comment:
Can you put this file in the `python/mastodonutils` folder instead and create unit tests for it? We have an existing `mastodonutils` python module and this would fit there perfectly fine. Also, please look at the `python/mastodonutils/test` folder for unit test examples.

Reply:
The GMSim.py needs to be in mastodon/data because the getDataFileNameByName function cannot find it in the mastodonutils